controllers/replace_misconfigured_pods.go

```diff
@@ -143,6 +108,54 @@ func (c ReplaceMisconfiguredPods) Reconcile(r *FoundationDBClusterReconciler, co
 	return nil
 }
 
+func instanceNeedsRemovalForPVC(pvc corev1.PersistentVolumeClaim, cluster  *fdbtypes.FoundationDBCluster) (bool, error) {
```

Comment:
```suggestion
func instanceNeedsRemovalForPVC( cluster  *fdbtypes.FoundationDBCluster, pvc corev1.PersistentVolumeClaim) (bool, error) {
```

All other methods take `cluster` as first argument, we should stick with that.